cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -163,6 +114,90 @@ if [ "${CURRENT_VERSION}" = "2.2.1" -a "${CURRENT_VERSION}" != "${TARGET_VERSION
   echo "Backup done in ${BACKUP_DIR}"
 fi
 
+# Rollback to previous minor version of etcd 3.x, if needed.
+# This approach has only been tested with 3.2 and earlier. If newer versions of etcd support this
+# downgrade approach, please update the "${CURRENT_MINOR_VERSION} -le ..." check here.
```

Comment:
I don't see this "-le" check.

Reply:
Good catch. I've removed this comment. We now have the tests, which ensures the rollback flow won't get broken without being noticed.